calculator.js

```diff
@@ -1,5 +1,113 @@
 const exampleAdditionInput = {
   num1: 3,
+  num2: 8,
+  operation: "add",
+}
+const exampleAdditionInput2 = {
+  num1: 11,
+  num2: 7,
+  operation: "+",
+}
+const exampleSubtractionInput = {
+  num1: 6,
   num2: 5,
-  operation: 'add',
+  operation: "subtract",
+}
+const exampleSubtractionInput2 = {
+  num1: 2,
+  num2: 12,
+  operation: "-",
+}
+const exampleMultiplicationInput = {
+  num1: 3,
+  num2: 1,
+  operation: "multiply",
+}
+const exampleMultiplicationInput2 = {
+  num1: 13,
+  num2: 2,
+  operation: "*",
+}
+const exampleDivisionInput = {
+  num1: 4,
+  num2: 2,
+  operation: "divide",
+}
+const exampleDivisionInput2 = {
+  num1: 15,
+  num2: 5,
+  operation: "/",
+}
+const exampleZeroDivisionInput = {
+  num1: 30,
+  num2: 0,
+  operation: "divide",
+}
+const exampleZeroDivisionInput2 = {
+  num1: 15,
+  num2: 0,
+  operation: "/",
+}
+const exampleNumberCheck = {
+  num1: "sushi",
+  num2: 13,
+  operation: "/",
+}
+const exampleOperatorCheck = {
+  num1: 2,
+  num2: 13,
+  operation: "?",
+}
+
+//hard code the operators
+const operatorOptions = ["add", "+", "subtract", "-", "multiply", "*", "divide", "/"];
+
+//prompts for user input
+console.log("Welcome to my first JavaScript calculator.");
+console.log("Please enter your two numbers, and one of the following operators; (+), subtract (-), multiply (*), or divide (/).");
+
+const validOperator = function validOperator(user_input) {
+  if(!operatorOptions.includes(user_input.operation)) {
+    console.log("Please enter a valid operator; +), subtract (-), multiply (*), or divide (/).");
+  }
+}
+const calculate = function(user_input) {
+  if (isNaN(user_input.num2) || isNaN(user_input.num1)){
```

Comment:
Consider encapsulating this functionality in it's own function, similar to `validOperator`